packages/helpers/src/phonebook.rs

```diff
@@ -0,0 +1,29 @@
+#[macro_export]
+macro_rules! get_contracts {
+    ($deps:expr, $factory_contract:expr, $($field_name:ident),*) => {
+        {
+            #[derive(Debug)]
+            struct Phonebook {
+                $(
+                    $field_name: String,
+                )*
+            }
+            let contracts = $deps
+                                .querier
+                                .query::<HashMap<String, String>>(&QueryRequest::Wasm(WasmQuery::Smart {
```

Comment:
This macro forces user to import the correct types manually, instead it would be much easier to refer to them directly from here:

```suggestion
                                .query::<std::collections::HashMap<String, String>>(&cosmwasm_std::QueryRequest::Wasm(cosmwasm_std::WasmQuery::Smart {
```

Reply:
You have fixed it for a HashMap, but not for QueryRequest and WasmQuery.

I think we better choose consistent design and either specify full path each and every time, or don't do that at all (though I still think a full path is better in this context).